pkg/commands/init.go

```diff
@@ -70,8 +70,24 @@ func NewInitCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			o.IOStreams.Info("App Deployed Succeed")
-			//TODO(wonderflow) Wait for app running, and print trait info such as route, domain
```

Comment:
by the way, I don't think this PR fixes this TODO

Reply:
Yes, I will put it back.